itez/beneficiary/views.py

```diff
@@ -382,16 +567,15 @@ def get_context_data(self, *args, **kwargs):
         services = {"services": []}
 
         latest_beneficiary_service = {
-            "service_name": latest_beneficiary_medical_record.service or "",
-            "service_facility": latest_beneficiary_medical_record.service_facility
-            or "",
-            "interaction_date": latest_beneficiary_medical_record.interaction_date
-            or "",
-            "service_provider": service_provider_name or "",
-            "service_provider_comments": latest_beneficiary_medical_record.provider_comments
-            or "",
-            "prescription": latest_beneficiary_medical_record.prescription.title or "",
-            "when_to_take": latest_beneficiary_medical_record.when_to_take or "",
+            "service_name": latest_beneficiary_medical_record.service,
+            "service_facility": latest_beneficiary_medical_record.service.service_type,
+            "interaction_date": latest_beneficiary_medical_record.interaction_date,
+            "service_provider": service_provider_name,
+            "service_provider_comments": latest_beneficiary_medical_record.provider_comments,
+            "supporting_documents": latest_beneficiary_medical_record.document,
+            "prescription": latest_beneficiary_medical_record.prescription.title,
+            "when_to_take": latest_beneficiary_medical_record.when_to_take,
+            "user_roles": user_roles() 
```

Comment:
take everything that is in upstream here and just append user_roles context variable here